sphinx/config.py

```diff
@@ -484,7 +484,7 @@ def check_unicode(config):
     nonascii_re = re.compile(br'[\x80-\xff]')
 
     for name, value in config._raw_config.items():
-        if isinstance(value, binary_type) and nonascii_re.search(value):
+        if isinstance(value, bytes) and nonascii_re.search(value):
             logger.warning(__('the config value %r is set to a string with non-ASCII '
                               'characters; this can lead to Unicode errors occurring. '
                               'Please use Unicode strings, e.g. %r.'), name, u'Content')
```

Comment:
I think this warning is not meaningful for py3.
This will be warned if users writes binary-string in conf.py like `b'Sphinx'`